lib/active_model_cachers/active_record/extension.rb

```diff
@@ -97,6 +97,8 @@ def define_callback_for_cleaning_cache(class_name, column, foreign_key, on: nil,
             changed = column ? previous_changes.key?(column) : previous_changes.present?
             clean.call(send(foreign_key)) if changed || destroyed?
           }, on: on
+
+          after_touch ->{ clean.call(send(foreign_key)) }
```

Comment:
I can't do this, because `touch` will clear changes: https://github.com/rails/rails/blob/3804d017333da16d76d9fc6633faf5635c7b03d7/activerecord/lib/active_record/persistence.rb#L472

That's mean `previous_changes ` will not include columns that changed via `touch`.

Reply:
Hmm, seems like `after_touch` doesn't provide a way that you know which columns are changed.